tools/ctor_evaller.py

```diff
@@ -125,7 +125,7 @@ def add_func(asm, func):
 var stackMax = stackTop + totalStack;
 if (stackMax >= totalMemory) throw 'not enough room for stack';
 
-var dynamicTop = stackMax;
+var dynamicTopPtr = stackMax;
```

Comment:
The key thing is that the value of dynamicTop should not change during the ctor evalling. Until now we did that by simply not allowing sbrk to run - it's just not present in the temp .js file we create, so it aborts if called. But with this PR, sbrk is in the asm.js module. So what you want to do is either remove it from the asm.js code, or put an `abort()` on its first line or something like that (the temp .js file we create and eval here isn't valid asm.js, so no need to worry about that).